script.js

```diff
@@ -1,4 +1,250 @@
+      "Welcome player " +
+      playerNumber +
+      "😀. " +
+      "<br>" +
+      "You rolled " +
+      randomNumberOne +
+      " for dice 1 and " +
+      randomNumberTwo +
+      " for dice 2.";
+
+    if (equals == false) {
+      outputString =
+        outputString +
+        "<br>" +
+        "You can now choose which dice goes first, dice 1 or dice 2. Please enter '1' or '2'" +
+        "<br>";
+    }
+  } // end of welcome message
+  else if (message == "leaderboard") {
+    outputString =
+      outputString +
+      "<br>" +
+      "============================================================" +
+      "<br>" +
+      "<br>" +
+      "🏆  <b><u>Leaderboard</u></b>  🏆" +
+      "<br>" +
+      "<br>";
+    if (
+      totalPlayerOneScore > totalPlayerTwoScore ||
+      totalPlayerOneScore == totalPlayerTwoScore
+    ) {
+      outputString =
+        outputString +
+        "Player One total Score = " +
+        totalPlayerOneScore +
+        "<br>" +
+        "Player Two total Score = " +
+        totalPlayerTwoScore +
+        "<br>";
+    } else {
+      outputString =
+        outputString +
+        "Player Two total Score = " +
+        totalPlayerTwoScore +
+        "<br>" +
+        "Player One total Score = " +
+        totalPlayerOneScore +
+        "<br>";
+    }
+  } else if (message == "click submit") {
+    outputString = "Please start the game by clicking submit.";
+  } // end of click submit message
+  else if (message == "result") {
+    if (equals == false) {
+      outputString = "Player " + playerNumber + ", you chose Dice " + input;
+    }
+
+    outputString =
+      outputString + "<br>" + "Your number is " + combinedNumber + "👍.";
+    if (playerNumber == 2) {
+      outputString =
+        outputString +
+        "<br>" +
+        winningPlayer +
+        " wins this round. Congrats!!🤝";
+    }
+    if (playerNumber == 1) {
+      outputString = outputString + " It is now Player 2's turn.";
+    }
+  } // end of result message
+  return outputString;
+};
+//function for first mode logic
+var rollDiceMode = function (input) {
+  if (input == "") {
+    randomNumberOne = rollADice();
+    randomNumberTwo = rollADice();
+
+    if (randomNumberOne == randomNumberTwo) {
+      equals = true;
+      combinedNumber = "" + randomNumberOne + randomNumberTwo;
+      if (playerNumber == 1) {
+        currentPlayerOneScore = Number(combinedNumber);
+        // playerOneScores.push(currentPlayerOneScore);
+        totalPlayerOneScore = totalPlayerOneScore + currentPlayerOneScore;
+      } else if (playerNumber == 2) {
+        currentPlayerTwoScore = Number(combinedNumber);
+        // playerTwoScores.push(currentPlayerTwoScore);
+        totalPlayerTwoScore = totalPlayerTwoScore + currentPlayerTwoScore;
+        if (currentPlayerOneScore > currentPlayerTwoScore) {
+          winningPlayer = "Player 1";
+        } else if (currentPlayerTwoScore > currentPlayerOneScore) {
+          winningPlayer = "Player 2";
+        }
```

Comment:
What will happen if both players draw?
You might want to include another condition for when both scores are the equal!